server/controllers/apply/applicationsController.ts

```diff
@@ -26,15 +26,24 @@ export default class ApplicationsController {
 
   index(): RequestHandler {
     return async (req: Request, res: Response) => {
-      const applications = await this.applicationService.getAllForLoggedInUser(req.user.token)
+      const inProgressApplications = await this.applicationService.getApplicationsBySubmissionStatusForLoggedInUser({
```

Comment:
I think this new param is intended to reduce volume of applications and calls made... 
would it be possible to just make one call to the API, and pass in the `isSubmitted` param from the UI path?
e.g. when I'm on the in progress tab we just make one call to the API with `isSubmitted` as false, and it just returns those applications. And vice versa for submitted tab

Reply:
I think CAS1 do something similar with their dashboard function the applications controller

Reply:
hmmmm, actually I don't think that is possible, because the hash fragment isn't available in the controller...